en_partut-ud-train.conllu

```diff
@@ -4707,7 +4707,7 @@
 20	fact	fact	NOUN	S	Number=Sing	15	nmod	_	_
 21	that	that	SCONJ	CS	_	32	mark	_	SpaceAfter=No
 22	,	,	PUNCT	FF	_	32	punct	_	_
-23	where	where	SCONJ	CS	_	29	mark	_	_
+23	where	where	ADV	B	PronType=Rel	29	advmod	_	_
```

Comment:
The idiom "where X is concerned". Borderline but I would lean against calling it locative, and thus not relative. "concerned" attaches as `advcl` not as a relative clause.